tools/scoring/tests/samples/proposed/entity_type_identification_reporting.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This file is used as a test fixture; it is a valid DBO instance
+# which demonstrates the features of a reporting entity.
+
```

Comment:
why not reuse the files you defined in the previous PR ?
are they the same content?

Reply:
They are presently but I would prefer to leave them separate given that they are testing separate features; that way, there are no hidden interdependencies and modifications will not have unintended results.

Reply:
Case in point: I have to change it to fix the test. :)

Reply:
I propose if they are the same reuse the same, once you need to change the content then fork them.